test/integration/test-classes/solution-explorer.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable @typescript-eslint/no-useless-constructor */
+/* eslint-disable no-useless-constructor */
+import path from 'path';
+
+import {Application} from 'spectron';
+import {GlobalMethods} from './global-methods';
+
+export class SolutionExplorer extends GlobalMethods {
+  constructor(app: Application) {
+    super(app);
+  }
+
+  public async show(): Promise<void> {
+    const solutionExplorerIsVisible = await this.webdriverClient.isVisible('[data-test-solution-explorer-panel]');
+    if (solutionExplorerIsVisible) {
+      return;
+    }
+
+    await this.ensureVisible('[data-test-toggle-solution-explorer]');
+    await this.clickOn('[data-test-toggle-solution-explorer]');
+    await this.ensureVisible('[data-test-solution-explorer-panel]');
+  }
+
+  public async hide(): Promise<void> {
+    const solutionExplorerIsVisible = await this.webdriverClient.isVisible('[data-test-solution-explorer-panel]');
```

Comment:
Du kannst hier eine Zeile sparen:

```suggestion
    const solutionExplorerIsHidden = !(await this.webdriverClient.isVisible('[data-test-solution-explorer-panel]'));
```